src/styles/_catalog.scss

```
@@ -1,22 +1,30 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(3, $card-width);
```

Let's use here max or min value and change your media queries a little bit according to it
```suggestion
  grid-template-columns: repeat(4, $card-width);
```
or 

```suggestion
  grid-template-columns: $card-width;
```

---

Thank you! Done.